src/main/scala/ai/privado/model/Constants.scala

```diff
@@ -167,4 +167,6 @@ object Constants {
 
   // Ruby defaults
   val defaultExpansionLimit = 100
+
+  val dummyFileName = "dummyFileName"
```

Comment:
Instead of calling this `dummyFileName` better we call it `unknown`, so this constant can be used across for all these kind of cases

Reply:
Agree, addressed here: c043eecc1ef51555209fc287a5bc7bc408f969ec